dotCMS/src/integration-test/java/com/dotcms/publishing/listener/SecurityKeyResetTest.java

```diff
@@ -0,0 +1,82 @@
+package com.dotcms.publishing.listener;
+
+import static com.dotcms.publisher.business.PublisherTestUtil.createEndpoint;
+import static com.dotcms.publisher.business.PublisherTestUtil.createEnvironment;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.LicenseTestUtil;
+import com.dotcms.company.CompanyAPI;
+import com.dotcms.company.CompanyAPIFactory;
+import com.dotcms.datagen.TestUserUtils;
+import com.dotcms.publisher.endpoint.bean.PublishingEndPoint;
+import com.dotcms.publisher.endpoint.business.PublishingEndPointAPI;
+import com.dotcms.publisher.environment.bean.Environment;
+import com.dotcms.publisher.pusher.PushPublisher;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.liferay.portal.model.Company;
+import com.liferay.portal.model.User;
+import com.liferay.util.Encryptor;
+import java.security.Key;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SecurityKeyResetTest extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)